src/mockApi/getData.js

```diff
@@ -0,0 +1,53 @@
+import { baseURL } from '../shared/constants';
+import { filterCurrentTasks, filterProgress } from '../shared/helpers';
+
+export async function getFakeUsers() {
+  const response = await fetch(`${baseURL}/users`);
+  const users = await response.json();
+
+  return users;
+}
+
+export async function getFakeTasks() {
+  const response = await fetch(`${baseURL}/tasks`);
+  const tasks = await response.json();
+
+  return tasks;
+}
+
+export async function getMemberTasks(id) {
+  const response = await fetch(`${baseURL}/users/?id=${id}`);
+  const user = await response.json();
+  const { tasksId } = user[0];
```

Comment:
why ?

Reply:
Because I get an array. Json server returns data like this.

console.log(user)
![image](https://user-images.githubusercontent.com/3121071/156301910-b48c7402-2e8c-4f25-9f81-4cf7a8ea4f08.png)